graphql/data/schema.js

```diff
@@ -948,6 +951,35 @@ const CauseType = new GraphQLObjectType({
   }),
   interfaces: [nodeInterface],
 })
+const SearchEngineType = new GraphQLObjectType({
+  name: SEARCH_ENGINE,
+  description: 'all important data for a search engine.',
+  fields: () => ({
+    id: globalIdField(SEARCH_ENGINE),
+    name: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: `Name of the Search Engine`,
+    },
+    searchUrl: {
+      type: new GraphQLNonNull(GraphQLString),
+      description:
+        'query string to redirect the user to after using the search bar',
+    },
+    rank: {
+      type: new GraphQLNonNull(GraphQLInt),
+      description: 'what order to display the search engine in a list',
+    },
+    isCharitable: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      description: `Whether or not the user can earn extra impact with this Search Engine`,
+    },
+    inputPrompt: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: `Display string to display in the search bar`,
+    },
+  }),
+  interfaces: [nodeInterface],
```

Comment:
You'll have to add this to the node interface definitions at [L155](https://github.com/gladly-team/tab/blob/355f35594439ccea639aedcc25ab91735d9a044b/graphql/data/schema.js#L155). This provides logic for how to resolve to an object from its global ID ([spec](https://relay.dev/graphql/objectidentification.htm) if you want some background reading).